src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/NomisApiService.kt

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.service
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.CreateRole
+
+@Service
+class NomisApiService(
+  @Qualifier("nomisWebClient") val nomisWebClient: WebClient
+) {
+  companion object {
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
+  @Throws(RoleExistsException::class)
+  fun createRole(createRole: CreateRole) {
+    AuthService.log.debug("Create dps role for $createRole.roleCode with {}", createRole)
```

Comment:
referencing different `log` here.  Also better for logging to use `{}` for all variables rather than having some interpolated